pkg/performanceprofile/controller/performanceprofile/components/tuned/tuned_test.go

```diff
@@ -112,11 +114,50 @@ var _ = Describe("Tuned", func() {
 			})
 
 			It("should contain high power consumption related parameters", func() {
+				profile.Spec.WorkloadHints.PerPodPowerManagement = pointer.BoolPtr(false)
 				manifest := getTunedManifest(profile)
 				By("Populating high power consumption cmdline")
 				Expect(cmdlineHighPowerConsumption.MatchString(manifest)).To(BeTrue())
 				By("Populating idle poll cmdline")
 				Expect(cmdlineHighPowerConsumptionWithRealtime.MatchString(manifest)).To(BeTrue())
+				By("Populating pstate cmdline")
+				Expect(cmdlineHighPowerConsumptionWithoutPerPodPowerManagement.MatchString(manifest)).To(BeTrue())
+			})
+		})
+
+		When("additional args", func() {
```

Comment:
The context here is a bit vague, what are you testing here?

Reply:
I'm testing when it has aadditional args field, it can be confused...

Reply:
This not correspond to this PR, better do it in a new one:
https://github.com/openshift/cluster-node-tuning-operator/pull/420